src/index.js

```diff
@@ -21,12 +21,11 @@ import { LoadingComponent } from './components/common';
 import { Profile } from './components/pages/Profile';
 
 import { createStore, applyMiddleware } from 'redux';
-import thunk from 'redux-thunk';
+
 import { Provider } from 'react-redux';
 import { appReducer } from './state/reducers/appReducer';
 
-export const store = createStore(appReducer, applyMiddleware(thunk));
-
+export const store = createStore(appReducer);
```

Comment:
I personally don't know the significance of removing, "applyMiddleware(thunk)" from the "const store = createStore(appReducer, applyMiddleware(thunk));"
This doesn't mean that it shouldn't be approved, I just don't know the significance.

Reply:
It was giving me a weird error and by the looks of it we will not need Async redux middleware  so it was redundant